### PR TITLE
[IMP] stock : Usability improvements

### PR DESCRIPTION
**Pull request by tivisse**
- Rename some labels
- Add menu tips for picking type views
- Set Operations menuitem in group_no_one
- The options under configuration should not be in technical features
- Reorganized the configuration menuitems
- Hide the internal transfers in the picking type views if the option Use multi location and warehouse is disabled

TASK : https://www.odoo.com/web#id=19393&view_type=form&model=project.task&action=333&active_id=248
PAD : https://pad.odoo.com/p/r.idXm0I0fuOHowMk2

### CODE REVIEW

File: addons/stock/stock_dashboard.xml [hunk 1]
```diff
@@ -34,6 +34,9 @@
                                         </button>
                                     </div>
                                     <div class="col-xs-6 o_kanban_primary_right">
+
```
Comment (tde-banana-odoo) [inline addons/stock/stock_dashboard.xml]: Unnecessary diff :) .

File: addons/stock/res_config.py [hunk 2]
```diff
@@ -80,10 +107,13 @@ class stock_config_settings(osv.osv_memory):
         'group_stock_tracking_owner': fields.boolean("Manage owner on stock",
             implied_group='stock.group_tracking_owner',
             help="""This way you can receive products attributed to a certain owner. """),
-        'group_stock_multiple_locations': fields.boolean("Manage multiple locations and warehouses",
+        'group_stock_multiple_locations': fields.boolean("Manage multiple locations",
             implied_group='stock.group_locations',
             help="""This will show you the locations and allows you to define multiple picking types and warehouses."""),
-        'group_stock_adv_location': fields.boolean("Manage advanced routes for your warehouse",
+        'group_stock_single_location': fields.boolean("Manage one unique location per warehouse",
```
Comment (tde-banana-odoo) [inline addons/stock/res_config.py]: "one unique", isn't it redundant ?

File: addons/stock/res_config.py [hunk 3]
```diff
@@ -80,10 +107,13 @@ class stock_config_settings(osv.osv_memory):
...
+            implied_group='stock.group_single_location',
+            help="""This will imply that you will manage one location per warehouse, i.e. no internal transfer is possible."""),
```
Comment (tde-banana-odoo) [inline addons/stock/res_config.py]: This implies that you manage.

File: addons/stock/security/stock_security.xml [hunk 4]
```diff
@@ -6,11 +6,17 @@
         <field name="name">Manage Multiple Locations and Warehouses</field>
         <field name="category_id" ref="base.module_category_hidden"/>
      </record>
+    <record id="group_single_location" model="res.groups">
+        <field name="name">Manage One Single Location per Warehouse</field>
```
Comment (tde-banana-odoo) [inline addons/stock/security/stock_security.xml]: id : res_groups_stock_single_location

Comment (tde-banana-odoo) [inline addons/stock/security/stock_security.xml]: Forget what I said.

File: addons/stock/stock_view.xml [hunk 5]
```diff
@@ -3,25 +3,24 @@
...
             sequence="30"/>
 
-        <menuitem id="menu_stock_warehouse_mgmt" name="Operations" parent="menu_stock_root" sequence="1"/>
+        <menuitem id="menu_stock_warehouse_mgmt" name="Operations" parent="menu_stock_root" sequence="2" groups="base.group_no_one"/>
         <menuitem id="menu_stock_config_settings" name="Configuration" parent="menu_stock_root"
             sequence="100" groups="group_stock_manager,base.group_configuration"/>
-        <menuitem id="menu_warehouse_config" name="Warehouse Management" parent="menu_stock_config_settings" sequence="1" groups="base.group_no_one"/>
-        <menuitem id="menu_reordering_rules_config" name="Schedulers Management" parent="menu_stock_config_settings" sequence="3" groups="base.group_no_one"/>
-        <menuitem id="menu_routes_config" name="Routes" parent="menu_stock_config_settings" sequence="2" groups="base.group_no_one"/>
+        <menuitem id="menu_warehouse_config" name="Warehouse Management" parent="menu_stock_config_settings" sequence="1"/>
```
Comment (tde-banana-odoo) [inline addons/stock/stock_view.xml]: Maybe reorder menu items by sequence, as you changed them ?

File: addons/stock/stock_view.xml [hunk 6]
```diff
@@ -875,10 +874,40 @@
...
+                Click here to create a new transfer.  
+              </p><p>
+                You can either do it immediatly or mark it as ToDo and then process it later. Use your scanner to validate the transferred quantity quicker.
+              </p>
+            </field>
         </record>
 
         <menuitem id="all_picking" name="All Transfers" parent="menu_stock_warehouse_mgmt" sequence="2" action="action_picking_tree_all" groups="stock.group_stock_manager,stock.group_stock_user"/>
 
+        <record id="action_picking_tree_picking_type" model="ir.actions.act_window">
```
Comment (tde-banana-odoo) [inline addons/stock/stock_view.xml]: id: stock_picking_action_picking_type

File: addons/stock/stock_view.xml [hunk 7]
```diff
@@ -1464,8 +1542,27 @@
             </field>
         </record>
 
-        <record id="action_picking_type_form" model="ir.actions.act_window">
-            <field name="name">Overview</field>
+        <record id="action_picking_type_form_single_location" model="ir.actions.act_window">
```
Comment (tde-banana-odoo) [inline addons/stock/stock_view.xml]: Update the id, following the convention.

File: addons/stock/stock_view.xml [hunk 8]
```diff
@@ -1464,8 +1542,27 @@
...
+              </p><p>
+                The picking type system allows you to assign each stock
+                operation a specific type which will alter its views accordingly.  
+                On the picking type you could e.g. specify if packing is needed by default, 
+                if it should show the customer.  
+              </p>
+            </field>
+        </record>
+
+        <record id="action_picking_type_form_multi_locations" model="ir.actions.act_window">
```
Comment (tde-banana-odoo) [inline addons/stock/stock_view.xml]: Update the id, following the convention.

File: addons/stock/stock_view.xml [hunk 9]
```diff
@@ -1024,6 +1095,13 @@
             }
             </field>
             <field name="search_view_id" ref="view_picking_internal_search"/>
+            <field name="help" type="html">
+              <p class="oe_view_nocontent_create">
+                Click here to create a new transfer.  
+              </p><p>
+                You can either do it immediatly or mark it as ToDo and then process it later. Use your scanner to validate the transferred quantity quicker.
```
Comment (tde-banana-odoo) [inline addons/stock/stock_view.xml]: ..." or mark it as Todo for future processing", maybe ? To propagate to other strings if modified.

**Comment by tivisse** [comment]
Landed in master at https://github.com/odoo/odoo/commit/e58b16a589307de6f5f6269a454fd6eeb813c3e2
